addons/web/static/src/core/confirmation_dialog/confirmation_dialog.js

```diff
@@ -95,6 +95,7 @@ export class AlertDialog extends ConfirmationDialog {
     static props = {
         ...ConfirmationDialog.props,
         contentClass: { type: String, optional: true },
+        getPayload: { type: Function, optional: true },
```

Comment:
move this somewhere else. I think this is only used in pos.

Reply:
We use directly that one in POS, if I move it, I need to create an entire override just to add an optional props